modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExecutionContext.java

```diff
@@ -223,6 +231,11 @@ public ClusterNode localNode() {
         return localNode;
     }
 
+    /** Default internal buffer size. */
+    public int defaultBufferSize() {
```

Comment:
the prefix `default-` implies there is also a non-default one in the context. I would suggest to rename it to simply `bufferSize` or `inBufferSize`